packages/custom-elements/tests/index.html

```diff
@@ -10,33 +10,28 @@
 -->
 <title>Custom Elements Tests</title>
 <meta charset="utf-8">
-<script>
-  (window.customElements = window.customElements || {}).forcePolyfill = true;
-</script>
-<script src="../node_modules/es6-promise/dist/es6-promise.auto.min.js"></script>
-<script src="../custom-elements.min.js"></script>
 <script src="../node_modules/wct-browser-legacy/browser.js"></script>
 
 <script>
   WCT.loadSuites([
-    'js/reactions.js',
-    'js/registry.js',
-    'js/instanceof.js',
-    'js/typescript.js',
-    'js/babel.js',
-    'js/closure.js',
-    'js/upgrade.js',
-    'js/shadow-dom.js',
-    'html/registry-upgrade.html',
-    'html/polyfillWrapFlushCallback/index.html',
-    'html/imports.html',
-    'html/shim.html',
-    'html/template-polyfill.html',
+    'html/reactions.html?__CE_installPolyfill=1',
+    'html/registry.html?__CE_installPolyfill=1',
+    'html/instanceof.html?__CE_installPolyfill=1',
+    'html/typescript.html?__CE_installPolyfill=1',
+    'html/babel.html?__CE_installPolyfill=1',
+    'html/closure.html?__CE_installPolyfill=1',
+    'html/upgrade.html?__CE_installPolyfill=1',
+    'html/shadow-dom.html?__CE_installPolyfill=1',
```

Comment:
I added a wrapper page to these tests so that they could be run as modules that import the workaround function.